### DISCUSSION TITLE
Preferred way to create a PNG

### DISCUSSION THREAD

**Discussion by dabrandstetter** [chillerlan/php-qrcode#261] [Q&A] [answered]
I am using version 5.0.2. and asking myself what is the best way to create a png qr code?

at the moment i use:

```
$options = new QROptions();
$options->outputType = QROutputInterface::CUSTOM;
$options->outputInterface = QRGdImagePNG::class;

(new QRCode($options))->render('data');
```

but `outputType` and `outputInterface` are deprecated.
am I doing something wrong?

**Comment by codemasher** [comment, marked as answer]
Hey, the deprecation is related to #223, which will remove/ignore the line `$options->outputType = QROutputInterface::CUSTOM;` in version 6 entirely.
Right now and as long as you're using version `^5.0` you will be fine.
